Add section on generate-metadata and generate-listing commands (#3721)
theforeman/foreman-documentation: guides/common/modules/proc_generating-metadata-and-listing-files-for-syncable-exports.adoc

> [id="generating-metadata-and-listing-files-for-syncable-exports_{context}"]

Reviewer: ```suggestion
[id="generating-metadata-and-listing-files-for-syncable-exports"]
```

suggestion: drop attribute in anchor.

For newly added files/anchors, we try to omit the "{context}" attribute unless required.

= Generating metadata and listing files for syncable exports

> In most cases, when exporting content in a syncable format using root privileges and the standard commands, metadata and listing files are generated automatically. However, you must manually trigger the generation of these files in the following circumstances:

Reviewer: ```suggestion
When exporting content in a syncable format, metadata and listing files are generated automatically.
However, you must manually trigger the generation of these files in the following circumstances:
```

* Please use one sentence per line.
* I suggest to drop "in most cases" because you have a list of all other cases below.
* I'd shorten it because we have to assume that users follow the docs.

* When a non-root user is doing the export.
* When the `--async` flag has been passed, for root or non-root user.

> * When triggering API manually or using Ansible modules to do the exports.

Reviewer: ```suggestion
* When using the {Project} API or Ansible modules to do the exports.
```

suggestion (non-blocking): minor rewording.

.Prerequisites
* The syncable export directory is accessible on disk.

.Procedure

> . To generate metadata for a syncable export that was performed asynchronously or if the metadata was lost, use the following command:

Reviewer: ```suggestion
. Generate metadata for a syncable export:
```

suggestion (non-blocking): reword

* I'd drop the "use the following command" because that is always the case.
* I'd shorten it because the "that" and "if" clause almost duplicates the information/list of reasons to do this in the beginning.

+
[options="nowrap" subs="+quotes"]
----

> # hammer content-export generate-metadata --path=/path/to/syncable/export

Reviewer: ```suggestion
$ hammer content-export generate-metadata --path=/path/to/syncable/export
```

----

> +

Reviewer: ```suggestion
```

not necessary as per asciidoc

> . To generate listing files for a syncable export directory, use the following command:

Reviewer: ```suggestion
. Generate listing files for a syncable export directory:
```

+
[options="nowrap" subs="+quotes"]
----

> # hammer content-export generate-listing --path=/path/to/syncable/export

Reviewer: ```suggestion
$ hammer content-export generate-listing --path=/path/to/syncable/export
```
